_stbt/mask.py

```diff
@@ -102,6 +102,24 @@ def __init__(self, m: MaskTypes, *, invert: bool = False) -> None:
             raise TypeError("Expected filename, Image, Mask, or Region. "
                             f"Got {m!r}")
 
+    @staticmethod
+    def from_alpha_channel(image: ImageT) -> Mask:
+        """Create a mask from the alpha channel of an image.
+
+        :type image: string or `numpy.ndarray`
+        :param image:
+          An image with an alpha (transparency) channel. This can be the
+          filename of a png file on disk, or an image previously loaded with
+          `stbt.load_image`.
+
+          Filenames should be relative paths. See `stbt.load_image` for the
+          path lookup algorithm.
+        """
+        image = load_image(image)
+        if image.shape[2] != 4:
+            raise ValueError(f"{image} doesn't have an alpha channel")
```

Comment:
Conceptually speaking we treat an image with `shape[2] < 4` as having an alpha channel of all `255`s.  So I think this should be:

```suggestion
            return Mask(_image_region(image))
```

That way calling code may not need to check `shape[2]` itself at all.  Interestingly then `Mask.from_alpha_channel` becomes a generalisation of `_image_region`.

Reply:
Done.